website/server/models/subscriptionPlan.js

```diff
@@ -16,6 +16,9 @@ export let schema = new mongoose.Schema({
   gemsBought: {type: Number, default: 0},
   mysteryItems: {type: Array, default: () => []},
   lastBillingDate: Date, // Used only for Amazon Payments to keep track of billing date
+  additionalData: mongoose.Schema.Types.Mixed,
```

Comment:
But for now only google payments is going to use it, right? I agree with @TheHollidayInn that it would be useful to document the data that can go in this object, we can update it if another payment method is going to use it